ui/src/app/pages/data/data-set/export-dataset-modal.tsx

```diff
@@ -69,6 +81,17 @@ export const ExportDatasetModal: (props: Props) => JSX.Element = fp.flow(withCur
       }
     }
 
+    function createExportDatasetRequest(): DataSetExportRequest {
+      return {
+        dataSetRequest: createDataSetRequest(),
```

Comment:
several of the properties here can use the shorter syntax where there are exact keyname matches:

```
{
  kernelType,
  notebookName,
  ...
```